LayoutTests/fast/dom/remove-top-layer-crash.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<body>
+    <script>
+        window.testRunner?.waitUntilDone();
+        window.testRunner?.dumpAsText();
+        let counter = localStorage.getItem('counter');
+        if (!counter)
+        counter = 1;
+        else
+        counter = +counter;
+        
+        (async () => {
+            const frameset = document.createElement('frameset');
+            document.body.appendChild(frameset);
+            frameset.popover = 'auto';
+            for (let i = 0; i < 50; i++) {
+                frameset.showPopover();
+                try {
+                    await navigator.wakeLock.request('screen');
+                } catch { }
+                try {
+                    await window.indexedDB.databases();
+                } catch { }
+                frames.document.body = frameset;
+            }
+        })();
+        
+        if (counter < 3) {
+            counter++;
+            localStorage.setItem('counter', counter);
+            location.reload();
+        } else window.testRunner?.notifyDone();
+    </script>
+    <p> Test PASS if no crash </p>
```

Comment:
```suggestion
    <p>PASS if test does not crashes</p>
```

The sentence is typically "PASS if test does not crashes"